src/schema/core/activemq-core-5.15.11-schema.html.asc.md5

```diff
@@ -0,0 +1,17 @@
+<html>
```

Comment:
This file is junk, its not an MD5 file but rather the HTML contents of a 404 error page. It doesn't seem like the MD5 of a signature is needed anyway.